app/app.go

```diff
@@ -130,6 +131,7 @@ var (
 
 		clptypes.ModuleName:     {authtypes.Burner, authtypes.Minter},
 		dispensation.ModuleName: {authtypes.Burner, authtypes.Minter},
+		ethbridge.ModuleName:    {authtypes.Burner, authtypes.Minter},
```

Comment:
Is this the right thing to do?  It definitely works, but it seems strange it wasn't here already

Reply:
Must've missed this one, ethbridge was one of the last modules we re-added once it was migrated.